Makefile

```diff
@@ -50,10 +50,10 @@ RELEASE_DEPS = libglibutil_release
 .PHONY: libglibutil_debug libglibutil_release
 
 libglibutil_debug:
-	make -C $(LIBGLIBUTIL_PATH) debug
+	$(MAKE) -C $(LIBGLIBUTIL_PATH) debug
```

Comment:
Is the order important in Makefiles? The default value for this variable is defined later in the file, same in unit/common/Makefile

Reply:
This code path is for local side-by-side builds (you can see the check for LIBGLIBUTIL_PATH above), did you do such a build during your test? Meaning that you have local build of libgliutil built and provide the path to that local build using LIBGLIBUTIL_PATH.

Reply:
Seems to work too, please test yourself of course too before merge (or we can move it closer to top if you prefer): https://pastebin.com/raw/T5ASCTL1

Reply:
AFAICT, in practice `MAKE` is always defined, `MAKE ?= make` is a no-op

Reply:
> AFAICT, in practice `MAKE` is always defined, `MAKE ?= make` is a no-op

It is (posix?) standard thing to define MAKE. So, for any conforming make that also supports (gnu extension?)  "?="  it would be a nop. Having an early failure due to syntax error or stte in case these conditions are not met by make du jour might serve some purpose though?

> Is the order important in Makefiles? The default value for this variable is defined later in the file, same in unit/common/Makefile

Depends - for example conditionals are interpreted in "as things are where the condition is"...

So IMO, if it is put somewhere, at top of a makefile would be better than at bottom.

Reply:
@monich  @spiiroin Make = make is part of [IEEE Std 1003.1-2017](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/make.html) section default rules subsection Macros.

Reply:
Also GNU make explains that $(MAKE) should always be used read:
https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html

Reply:
Thanks, I think this should be good to go now :)